src/formBuilder/FormBuilder.test.js

```diff
@@ -212,4 +212,55 @@ describe('FormBuilder', () => {
     expect(cardInputs.at(1).props().value).toEqual('Custom Title');
     expect(cardInputs.at(2).props().value).toEqual('Custom Description');
   });
+
+  it('supports the $schema keyword and there is no error', () => {
+    const jsonSchema = {
+      $schema: 'http://json-schema.org/draft-07/schema#',
+    };
+
+    const props = {
+      schema: JSON.stringify(jsonSchema),
+      uiSchema: '{}',
+      onChange: jest.fn(() => {}),
+      mods: {},
+      className: 'my-form-builder',
+    };
+
+    const div = document.createElement('div');
+    document.body.appendChild(div);
+    const wrapper = mount(<FormBuilder {...props} />, { attachTo: div });
```

Comment:
do we have to render something just to test that a property is supported? Is there no other validation function for schema validation?

Reply:
You're right, there are unit tests that I can write in addition to these :)

Reply:
I've added some unit tests too :) ✅